source/seismology/introduction.rst

```diff
@@ -1,2 +1,88 @@
 简介
 ====
+
+:本节贡献者: |姚家园|\（作者）、
+             |田冬冬|\（作者）
+:最近更新日期: 2021-04-23
+:预计阅读时间: 10 分钟
+
+----
+
+地震学是什么？
+--------------
+
+Keiiti Aki 和 Paul G. Richards 编写的经典地震学教材
+《Quantitative Seismology: Theory and Methods》对地震学的定义如下：
+
+  **Seismology is a science based on data called seismograms,
+  which are records of mechanical vibrations of the Earth.**
+
+  地震学是一门基于记录着地球机械振动的地震图的科学。
+
+  **These vibrations may be caused artificially by man-made explosions,
+  or they may be caused naturally by earthquakes and volcanic eruptions.**
+
+  振动可能是人工爆炸造成的，也可能是地震和火山喷发造成的。
+
+可以看出，地震学的研究工具是地震图，即地震仪记录的地球机械振动，而产生这些振动的源
+（一般称为\ **震源**\ ）可能是人工爆炸、地震和火山喷发等。震源激发的地球机械振动
+（一般称为\ **地震波**\ ）传播到地球内部，并最终被布设在地表或地下的地震仪
+（一般称为\ **地震台站**\ 或简称\ **台站**\ ） 记录下来。
+
+1994 年 1 月 17 日，美国加利福尼亚州 Northridge 附近发生了 `6.7 级地震 <https://earthquake.usgs.gov/earthquakes/eventpage/ci3144585/>`__，
+其释放出的能量相当于将近 20 亿千克高爆炸药。该地震产生的地震波经过地球内部传播后，
+被世界各地的地震台站记录到。一些地震波的传播路径以及全球台站记录的地表振动如下图所示。
+
+.. figure:: seismic-waves.jpg
+   :alt: 地震激发的地震波路径和波形
+   :width: 95%
+   :align: center
+
+   引自 `Exploring the Earth Using Seismology <https://www.iris.edu/hq/inclass/fact-sheet/exploring_earth_using_seismology>`__
+
+因此，地震图携带了震源和地球内部结构的信息，地震学家正是通过分析地震图来研究产生这些振动的
+震源以及地球内部结构。地震学研究的基本原理则是地震波理论，包括震源和波传播两个部分，
+前者主要关于震源如何激发地震波，后者主要有关地震波如何在地下传播。
+
+地震学与地震
+------------
+
+地震学的英文单词是 Seismology，地震的英文单词是 Earthquake。由于中文翻译的问题，
+大众或非地震学专业的学者可能会混淆地震学和地震这两个概念，认为地震学就是关于地震的科学。
+从上文的定义可以看出，地震只是地震学的一个重要部分。任何产生能被地震仪记录到的地球机械
+振动的源都可以是地震学的研究对象，比如人工爆炸、火山喷发、人类活动、风等。
```

Comment:
It would be better to add more examples here, like ocean waves, hurricane, traffic et al.

Reply:
Sure. I'll add more in the introduction to seismology. The current version may only cover 50-70%.